from PyQt5.QtWidgets import QMainWindow, QApplication, QMessageBox
from PyQt5 import QtCore, QtGui, QtWidgets
from controller.UsuarioCTR import UsuarioCTR

class Ui_FormUsuarios(object):

    def setupUi(self, FormUsuarios):
        FormUsuarios.setObjectName("FormUsuarios")
        FormUsuarios.resize(661, 471)
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap("cliente.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        FormUsuarios.setWindowIcon(icon)
        self.centralwidget = QtWidgets.QWidget(FormUsuarios)
        self.centralwidget.setObjectName("centralwidget")
        self.label = QtWidgets.QLabel(self.centralwidget)
        self.label.setGeometry(QtCore.QRect(30, 20, 51, 21))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.label.setFont(font)
        self.label.setObjectName("label")
        self.editPesquisaUsuario = QtWidgets.QLineEdit(self.centralwidget)
        self.editPesquisaUsuario.setGeometry(QtCore.QRect(90, 20, 371, 21))
        self.editPesquisaUsuario.setText("")
        self.editPesquisaUsuario.setObjectName("editPesquisaUsuario")
        self.tableCliente = QtWidgets.QTableWidget(self.centralwidget)
        self.tableCliente.setGeometry(QtCore.QRect(180, 60, 321, 301))
        self.tableCliente.setLayoutDirection(QtCore.Qt.LeftToRight)
        self.tableCliente.setAutoFillBackground(False)
        self.tableCliente.setEditTriggers(QtWidgets.QAbstractItemView.NoEditTriggers)
        self.tableCliente.setGridStyle(QtCore.Qt.SolidLine)
        self.tableCliente.setRowCount(0)
        self.tableCliente.setObjectName("tableCliente")
        self.tableCliente.setColumnCount(2)
        item = QtWidgets.QTableWidgetItem()
        item.setTextAlignment(QtCore.Qt.AlignCenter)
        font = QtGui.QFont()
        font.setBold(True)
        font.setWeight(75)
        item.setFont(font)
        self.tableCliente.setHorizontalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        item.setTextAlignment(QtCore.Qt.AlignCenter)
        font = QtGui.QFont()
        font.setBold(True)
        font.setWeight(75)
        item.setFont(font)
        self.tableCliente.setHorizontalHeaderItem(1, item)
        self.layoutWidget = QtWidgets.QWidget(self.centralwidget)
        self.layoutWidget.setGeometry(QtCore.QRect(470, 15, 166, 31))
        self.layoutWidget.setObjectName("layoutWidget")
        self.horizontalLayout = QtWidgets.QHBoxLayout(self.layoutWidget)
        self.horizontalLayout.setContentsMargins(0, 0, 0, 0)
        self.horizontalLayout.setObjectName("horizontalLayout")
        self.btnPesquisarUsuario = QtWidgets.QPushButton(self.layoutWidget)
        self.btnPesquisarUsuario.setMaximumSize(QtCore.QSize(100, 40))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.btnPesquisarUsuario.setFont(font)
        icon1 = QtGui.QIcon()
        icon1.addPixmap(QtGui.QPixmap("lupa.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.btnPesquisarUsuario.setIcon(icon1)
        self.btnPesquisarUsuario.setIconSize(QtCore.QSize(16, 16))
        self.btnPesquisarUsuario.setObjectName("btnPesquisarUsuario")
        self.horizontalLayout.addWidget(self.btnPesquisarUsuario)
        self.btnLimpar = QtWidgets.QPushButton(self.layoutWidget)
        self.btnLimpar.setMaximumSize(QtCore.QSize(100, 40))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.btnLimpar.setFont(font)
        icon2 = QtGui.QIcon()
        icon2.addPixmap(QtGui.QPixmap("borracha.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.btnLimpar.setIcon(icon2)
        self.btnLimpar.setIconSize(QtCore.QSize(16, 16))
        self.btnLimpar.setObjectName("btnLimpar")
        self.horizontalLayout.addWidget(self.btnLimpar)
        self.layoutWidget1 = QtWidgets.QWidget(self.centralwidget)
        self.layoutWidget1.setGeometry(QtCore.QRect(120, 370, 415, 42))
        self.layoutWidget1.setObjectName("layoutWidget1")
        self.horizontalLayout_2 = QtWidgets.QHBoxLayout(self.layoutWidget1)
        self.horizontalLayout_2.setContentsMargins(0, 0, 0, 0)
        self.horizontalLayout_2.setObjectName("horizontalLayout_2")
        self.btnExcluirUsuario = QtWidgets.QPushButton(self.layoutWidget1)
        font = QtGui.QFont()
        font.setPointSize(10)
        self.btnExcluirUsuario.setFont(font)
        icon3 = QtGui.QIcon()
        icon3.addPixmap(QtGui.QPixmap("excluircliente.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.btnExcluirUsuario.setIcon(icon3)
        self.btnExcluirUsuario.setIconSize(QtCore.QSize(30, 30))
        self.btnExcluirUsuario.setObjectName("btnExcluirUsuario")
        self.horizontalLayout_2.addWidget(self.btnExcluirUsuario)
        self.gridLayout = QtWidgets.QGridLayout()
        self.gridLayout.setObjectName("gridLayout")
        self.btnAlterarUsuario = QtWidgets.QPushButton(self.layoutWidget1)
        font = QtGui.QFont()
        font.setPointSize(10)
        self.btnAlterarUsuario.setFont(font)
        icon4 = QtGui.QIcon()
        icon4.addPixmap(QtGui.QPixmap("editar usuario.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.btnAlterarUsuario.setIcon(icon4)
        self.btnAlterarUsuario.setIconSize(QtCore.QSize(30, 30))
        self.btnAlterarUsuario.setObjectName("btnAlterarUsuario")
        self.gridLayout.addWidget(self.btnAlterarUsuario, 0, 0, 1, 1)
        self.btnCadastrarUsuario = QtWidgets.QPushButton(self.layoutWidget1)
        font = QtGui.QFont()
        font.setPointSize(10)
        self.btnCadastrarUsuario.setFont(font)
        icon5 = QtGui.QIcon()
        icon5.addPixmap(QtGui.QPixmap("cadastrar usuario.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.btnCadastrarUsuario.setIcon(icon5)
        self.btnCadastrarUsuario.setIconSize(QtCore.QSize(30, 30))
        self.btnCadastrarUsuario.setObjectName("btnCadastrarUsuario")
        self.gridLayout.addWidget(self.btnCadastrarUsuario, 0, 1, 1, 1)
        self.horizontalLayout_2.addLayout(self.gridLayout)
        FormUsuarios.setCentralWidget(self.centralwidget)
        self.statusbar = QtWidgets.QStatusBar(FormUsuarios)
        self.statusbar.setObjectName("statusbar")
        FormUsuarios.setStatusBar(self.statusbar)

        self.retranslateUi(FormUsuarios)
        self.btnLimpar.clicked.connect(self.editPesquisaUsuario.clear)
        QtCore.QMetaObject.connectSlotsByName(FormUsuarios)
        FormUsuarios.setTabOrder(self.editPesquisaUsuario, self.btnPesquisarUsuario)
        FormUsuarios.setTabOrder(self.btnPesquisarUsuario, self.btnLimpar)
        FormUsuarios.setTabOrder(self.btnLimpar, self.tableCliente)
        FormUsuarios.setTabOrder(self.tableCliente, self.btnAlterarUsuario)
        FormUsuarios.setTabOrder(self.btnAlterarUsuario, self.btnCadastrarUsuario)

    def retranslateUi(self, FormUsuarios):
        _translate = QtCore.QCoreApplication.translate
        FormUsuarios.setWindowTitle(_translate("FormUsuarios", "Usuários"))
        self.label.setText(_translate("FormUsuarios", "Usuário:"))
        item = self.tableCliente.horizontalHeaderItem(0)
        item.setText(_translate("FormUsuarios", "Cód. Usuário"))
        item = self.tableCliente.horizontalHeaderItem(1)
        item.setText(_translate("FormUsuarios", "Usuário"))
        self.btnPesquisarUsuario.setText(_translate("FormUsuarios", "Pesquisar"))
        self.btnLimpar.setText(_translate("FormUsuarios", "Limpar"))
        self.btnExcluirUsuario.setText(_translate("FormUsuarios", "Excluir Usuário"))
        self.btnAlterarUsuario.setText(_translate("FormUsuarios", "Alterar Usuário"))
        self.btnCadastrarUsuario.setText(_translate("FormUsuarios", "Cadastrar Usuário"))

if __name__ == '__main__':
    import sys
    app = QApplication(sys.argv)
    MainWindow = QMainWindow()
    ui = Ui_FormUsuarios()
    ui.setupUi(MainWindow)
    MainWindow.show()
    sys.exit(app.exec_())